### ISSUE TITLE
Archived photo does not become visible when uploaded again

### ISSUE THREAD

**Issue by tmb80c** [photoprism/photoprism#250]
May be a bug:

I uploaded a photo to the demo. Then I archived it and uploaded it again (with changed metadata).

The picture does not become visible. Looks like it is treated as a duplicate which has already been deleted / archived.

Same behaviour in my local instance (latest image from Feb 7).

**Comment by lastzero** [comment]
Yes, will be treated as a duplicate. You want it to become visible again?

**Comment by tmb80c** [comment]
> 
> 
> Yes, will be treated as a duplicate. You want it to become visible again?

That's tricky I understand.

In the case of reindexing of Originals someone just wants to ensure that only new (manual linked / copied) photos will be added / indexed while archieved photos shall not show up again. I would prefer a check box where I can decide: only new added files in Originals shall be reindexed and archives remain archives or if checkbox is clicked all archieved (if not deleted in manually in Originals) will be brought back. Or there could be a list in libraries where I can list all archieved to have a chance to bring single photos back. 

For the upload and the import it sounds to me more logical that someone can add / replace a photo again if it has been erroneously archieved or the same photo with a new exposure or keywords shall be added.

**Comment by lastzero** [comment]
So basically one more checkbox for import / upload needed :)

When indexing originals, you already have the choice to index all or just new photos. Doesn't remove the archive flag though.

**Comment by graciousgrey** [comment]
I guess this https://github.com/photoprism/photoprism/issues/1308 could be a solution. Once it is implemented the archived file and the new version would be displayed as duplicates and you could decide to delete the archived one so that the new one is displayed.

**Comment by lastzero** [comment]
I'll go ahead and close this issue since we haven't received any new feedback or contributions for it.
